docs/runbook/Features.md

```diff
@@ -173,90 +98,168 @@ on the rooftop is selected, which can be difficult to do manually.
 
-## Maximum Common Bandwidth (MCB)
+## POP Placement
 
-Maximum Common Bandwidth (MCB) equally distributes bandwidth across all connected
-clients. When MCB is disabled, the total shortage in the network is minimized
-but there are no guarantees of how that shortage is distributed among clients.
-In fact, when MCB is disabled, if the input demand is too high for the
-underlying network, some clients can be disconnected in order to improve
-the bandwidth to the connected clients. Instead, MCB ensures that all of
-the clients that can be connected are connected, although sometimes at
-a lower bandwidth. This is roughly equivalent to lowering the demand.
-
-The minimum guaranteed bandwidth can be improved by relieving network congestion
-of highly utilized links either by moving POPs or placing extra POPs. See the
-[Pop Placement](#pop-placement) feature for information on how to quickly place
-extra POPs.
-
-**How to use it?**
-
-1.  Go to the Network Design section.
-2.  Set the Maximize minimum guaranteed bandwidth parameter to "True".
-3.  Set the Link Capacity Filter value (Gbps) as required; Default = 0
-
-**Note:** Some links might have 0 capacity. In that case, if such a link is
-necessary to reach the site, it\'s still unreachable. There is also a
-parameter to filter links below some capacity. That could be useful in some
-rare cases where low capacity links are connecting some sites causing the max
-common bandwidth to be low. In that case, it's better to disconnect those sites
-by filtering out the low capacity links (that is, prevent the optimizer from
-selecting them). Regardless, if you choose to filter links below 10Mbps for
-example, then all such links will be marked unreachable.
-
-**Key Metrics**
-
-- Number of active CN/DN/POPs connected to demand sites
-- Number of active/connectable candidate/total candidate CN sites
-- Full topology flow results - Percent of demand sites, minimum guaranteed
-  bandwidth (Gbps)
+POP Placement allows you to specify additional POPs you would like to add to a
+candidate topology. The POPs are selected from the candidate DNs.
 
-## Tiered Service
+This can be useful to meet CIR requirements when the number of provided
+POPs is insufficient. It can also help connect disconnected portions of the
+network. It is particularlly useful in early-stage planning where not all
+possible POP locations are already known.
 
-Previously the TG Planner only provided identical bandwidth to each of the CNs
-it serves. With the Tiered Service feature, you can now provide different levels
-of bandwidth to a selected number of CNs. The Tiered Service feature properly
-addresses two use cases:
+**How to use it**
 
-1.  Multi-dwelling buildings with units requiring different levels of bandwidth.
-2.  Network shared by businesses and residential homes where businesses require
-    a higher amount of bandwidth.
+1. Specify the number of additional POPs you would like in `NUMBER_OF_EXTRA_POPS`
```

Comment:
You are right. I was deciding between using numbers for this subsection every time or omitting the number for a single step. I went for the consistent style but since it looks strange, I'll take your suggestion.